orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/TrafficGuard.java

```diff
@@ -163,6 +163,16 @@ public void verifyTrafficRemoval(
       Location location,
       String cloudProvider,
       String operationDescriptor) {
+
+    if (!hasDisableLock(serverGroupMoniker, account, location)) {
```

Comment:
will this condition prevent throwing `TrafficGuardException` if traffic guard is not configured? (not sure what `hasDisableLock` is doing)

Reply:
oh I see, this call is calling `front50` to verify it.

Reply:
Yes, `hasDisableLock` checks the trafficGuard section of the app config in front50. No traffic guard configured, no exception